src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should have right length', () => {
+    const result = getHumanAge(2, 3);
+
+    expect(result).toHaveLength(2);
+  });
+
+  test('should return an Array', () => {
+    const result = getHumanAge;
+
+    expect(Array.isArray(result));
```

Comment:
The test for checking if the result is an array is incorrect. The `getHumanAge` function should be called to get the result, and then `Array.isArray(result)` should be used in an expectation, like `expect(Array.isArray(result)).toBe(true);`.